generators/app/templates/storage/src/index.js

```diff
@@ -0,0 +1,35 @@
+class VerdaccioStoragePlugin {
```

Comment:
Do you want to defer Typescript for later?

Reply:
to templates?
ahh....I think I would prefer giving a options as to `language preferred` in the prompts and then simply taking those values whiling copying like this

```js
{
  name: 'lang',
}

// and in copying
 this.fs.copy(
      this.templatePath(`${this.props.pluginType}/src/index.${this.props.lang}`),
      this.destinationPath(resolve(this.destinationPathName, `src/index.${this.props.lang}`)),
      this.props
    );

```

But this can then need to add a different common folder as it need to setup typescript environment also

maybe we can have two common templates

- common
- ts-common

Reply:
Ok, seems like double work, in my experience helping people working without types at Verdaccio they struggle a lot with the interfaces they must implement, the big majority always clap over using TS, thus, for me is ok have two templates, but I'd like have TS as default.

Reply:
If you want I will submit a one more PR with both the templates.

Reply:
It make sense tome :-) feel free to merge this one.